superset-frontend/src/dashboard/util/getComponentWidthFromDrop.js

```diff
@@ -50,7 +50,7 @@ export default function getComponentWidthFromDrop({
   let destinationCapacity =
     destinationWidth.width - destinationWidth.occupiedWidth;
 
-  if (Number.isNaN(destinationCapacity)) {
+  if (Number.isNaN(Number(destinationCapacity))) {
```

Comment:
Can we move these `Number` coercing above to where the variables are defined?

Reply:
It looks to me that you still want to return the original value as defined if it's not a number. If I coerce them earlier, you're going to get NaN as a value. I'm reading through the code now to see if that will work, but you're probably more familiar with the code than I am.

Reply:
I actually know nothing about this file, lol. Just took another look, for `destinationCapacity` at least, it's calculated from an arithmetic operation so it's definitely expected to be a number.

Reply:
`destinationWidth.draggingWidth` is the only that still returns a value even though it may not be a number.